packages/graphql-codegen-cli/src/utils/watcher.ts

```diff
@@ -66,7 +66,7 @@ export const createWatcher = (
       parcelWatcher = await import('@parcel/watcher');
     } catch (err) {
       log(
-        `Parcel watcher not found. To use this feature, please make sure to provide @parcel/watcher as a peer dependency.`
+        `Failed to import @parcel/watcher, a required dependency for watch mode, due to the following error:\n${err}`
```

Comment:
Maybe we should also link to the documentation or print information on how to install the dependency?

Reply:
Sure, I updated both the error message and the docs with a link to https://www.npmjs.com/package/@parcel/watcher.